src/cdk/drag-drop/directives/drag.ts

```diff
@@ -392,6 +399,29 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
         }
       }
     });
+
+    // This only needs to be resolved once.
+    ref.beforeStarted.pipe(take(1)).subscribe(() => {
+      // If we managed to resolve a parent through DI, use it.
+      if (this._parentDrag) {
+        ref.withParent(this._parentDrag._dragRef);
+        return;
+      }
+
+      // Otherwise fall back to resolving the parent by looking up the DOM. This can happen if
+      // the item was projected into another item by something like `ngTemplateOutlet`.
+      let parent = this.element.nativeElement.parentElement;
+      while (parent) {
+        // `classList` needs to be null checked, because IE doesn't add it on some elements.
+        if (parent.classList && parent.classList.contains(DRAG_HOST_CLASS)) {
```

Comment:
```suggestion
        if (parent.classList?.contains(DRAG_HOST_CLASS)) {
```
Nullsafe dereference?